Fix host.json examples (#65243)
Azure/azure-content: includes/functions-host-json-durabletask.md

Configuration settings for [Durable Functions](../articles/azure-functions/durable-functions-overview.md).

> [!NOTE]

> > The below examples are for use with Azure Functions V2 or V3. If using Azure Functions V1, the "durableTask" section of the host.json should go in the root, rather than as a field under "extensions".

Reviewer: Here's a suggestion. It's wordy, but might help provide a bit more context.

```suggestion
> All major versions of Durable Functions are supported on all versions of the Azure Functions runtime. However, the schema of the host.json configuration is slightly different depending on the version of the Azure Functions runtime as well as the Durable Functions extension version being used. The below examples are for use with Azure Functions 2.0 and 3.0. in both examples, if using Azure Functions 1.0, the available settings are the same but the "durableTask" section of the host.json should go in the root of the host.json configuration, rather than as a field under "extensions".
```

> ### Durable Functions 1.x

Reviewer: I wonder if we should put the Durable Functions 2.x section on top since that's generally the version we want people using.

```json
{

>   "extensions" {

Author: ```suggestion
  "extensions": {
```

    "durableTask": {
      "hubName": "MyTaskHub",
      "controlQueueBatchSize": 32,
      "partitionCount": 4,
      "controlQueueVisibilityTimeout": "00:05:00",
      "workItemQueueVisibilityTimeout": "00:05:00",
      "maxConcurrentActivityFunctions": 10,
      "maxConcurrentOrchestratorFunctions": 10,
      "maxQueuePollingInterval": "00:00:30",
      "azureStorageConnectionStringName": "AzureWebJobsStorage",
      "trackingStoreConnectionStringName": "TrackingStorage",
      "trackingStoreNamePrefix": "DurableTask",
      "traceInputsAndOutputs": false,
      "logReplayEvents": false,
      "eventGridTopicEndpoint": "https://topic_name.westus2-1.eventgrid.azure.net/api/events",
      "eventGridKeySettingName":  "EventGridKey",
      "eventGridPublishRetryCount": 3,
      "eventGridPublishRetryInterval": "00:00:30",
      "eventGridPublishEventTypes": ["Started", "Completed", "Failed", "Terminated"]

>   }

Author: ```suggestion
    }
```
